.github/workflows/gql-perf-arch-github_workflow.yml

```diff
@@ -0,0 +1,70 @@
+name: gql-perf-arch-github_workflow
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      COMMIT_HASH_INPUT:
+        description: 'Commit hash'
+        required: true
+      CLASSES_TO_EXECUTE_INPUT:
+        description: 'Classes to test'
+        required: false
+      PULL_REQUEST_NUMBER_INPUT:
+        description: 'Pull request number'
```

Comment:
can you the usage of each of these parameters to the description? for example `if this is specified, we will comment a link to the tests results on this pull request`.

Reply:
I added it to the description.